src/main/haskell/kore/src/Kore/IndexedModule/IndexedModule.hs

```diff
@@ -724,3 +730,24 @@ hookedObjectSymbolSentences
     Map.restrictKeys
         indexedModuleObjectSymbolSentences
         indexedModuleHookedIdentifiers
+
+indexedModuleSubsorts
+    :: IndexedModule sortParam pat variables atts
+    -> [Subsort]
+indexedModuleSubsorts imod =
+    case internalIndexedModuleSubsorts imod of
+        Right subsorts -> subsorts
+        Left err -> error $ "IndexedModule should already have checked"
+                         ++ "form of subsort attributes, but parsing failed\n:"
```

Comment:
Indentation should be a multiple of 4. Also, increasing the indentation level by more than one level creates code that, to me, looks unbalanced, and is slightly harder to change, but that's a matter of taste and probably not everyone agrees.